.gitignore

```diff
@@ -1,6 +1,7 @@
 lib/*
 dist/*
 node_modules/*
+typings/*
```

Comment:
This folder isn't generated now with `@types` and it was removed from `.gitignore`. It's fine to have here but if you delete the folder locally that should be good enough.